src/src_V2/components/authInfoWrapper/index.jsx

```diff
@@ -0,0 +1,12 @@
+import styles from './styles.module.css';
+import logo from '../../assets/Travely.png';
+
+export default function AuthInfoWrapper({ ...props }) {
+  return (
+    <div className={styles.wrapper}>
```

Comment:
pls use Box

Reply:
Although we use MUI, but without using the functionality of the Box I think there is no need to replace the div with the Box. After all it's a component with it's own logic.